examples/trading/frontend/src/components/escrows/CreateEscrow.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { ApiLockedObject } from "@/types/types";
+import { useCurrentAccount, useSuiClientInfiniteQuery } from "@mysten/dapp-kit";
+import { formatAddress } from "@mysten/sui.js/utils";
+import { Avatar, Button, Select } from "@radix-ui/themes";
+import { InfiniteScrollArea } from "@/components/InfiniteScrollArea";
+import { useState } from "react";
+import { ObjectLink } from "../ObjectLink";
+import { useCreateEscrowMutation } from "@/mutations/escrow";
+
+export function CreateEscrow({ locked }: { locked: ApiLockedObject }) {
+  const [objectId, setObjectId] = useState<string | undefined>(undefined);
+  const account = useCurrentAccount();
+
+  const { mutate: createEscrowMutation, isPending } = useCreateEscrowMutation();
+
+  const { data, fetchNextPage, isFetchingNextPage, hasNextPage, refetch } =
+    useSuiClientInfiniteQuery(
+      "getOwnedObjects",
+      {
+        owner: account?.address!,
+        options: {
+          showDisplay: true,
+          showType: true,
+        },
+      },
+      {
+        enabled: !!account,
+        select: (data) =>
+          data.pages
+            .flatMap((page) => page.data)
+            .filter(
+              (x) => !!x.data?.display && !!x.data?.display?.data?.image_url,
```

Comment:
Worth calling out that this is basically just for demo purposes -- there's no reason why someone can't escrow an object that doesn't have `Display` or whose Display doesn't have an `image_url`.